src/wallet/walletdb.cpp

```diff
@@ -599,6 +599,16 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 return false;
             }
         }
+        else if (strType == "hdchain")
```

Comment:
Yes. I did refactor out these stings into constants serval times. But this should be done in another PR and hopefully after this has been merged.